packages/core/contracts/common/implementation/ExpandedERC20ExclusiveMinter.sol

```diff
@@ -0,0 +1,70 @@
+pragma solidity ^0.6.0;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "./MultiRole.sol";
+import "../interfaces/ExpandedIERC20ExclusiveMinter.sol";
+
+
+/**
+ * @title Modifies the ExpandedERC20 contract by changing the minter role to an exclusive one.
+ */
+contract ExpandedERC20ExclusiveMinter is ExpandedIERC20ExclusiveMinter, ERC20, MultiRole {
```

Comment:
This contract copies `ExpandedERC20` exactly but adds an `isMinterExclusive` method. The reason why `mint` and `burn` need to be re-implemented is that this contract cannot extend `ExpandedERC20` because the constructor needs to be re-written